product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/util/SCIMProvisioningUtil.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.identity.scenarios.commons.util;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.json.simple.JSONObject;
+import java.io.IOException;
+
+import static org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil.constructBasicAuthzHeader;
+import static org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil.sendPostRequestWithJSON;
+import static org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil.sendDeleteRequest;
+/**
+ *  Utility class for common functions for user management scenario.
+ */
+public class SCIMProvisioningUtil {
+
+
+    /**Headers
```

Comment:
Move `Headers` to the next line. Also let's change the method name `getCommonHeaders`

Reply:
done, updated the relevant areas